awx/settings/development.py

```diff
@@ -53,6 +53,10 @@
 
 # celery is annoyingly loud when docker containers start
 LOGGING['loggers'].pop('celery', None)  # noqa
+# avoid awx.main.dispatch WARNING-level scaling worker up/down messages
+LOGGING['loggers']['awx.main.dispatch']['level'] = 'ERROR'  # noqa
```

Comment:
This has a pretty severely silencing effect on the dispatcher logs, and I'm interested to see if I can return to it to restore those messages.